test/utils/ui_state/titration/initial_titration_test.py

```diff
@@ -132,7 +131,7 @@ def test_initial_titration_automatic(print_mock, set_next_state_mock):
         [
             mock.call("Heating to 30 C...", line=1),
             mock.call("", line=2),
-            mock.call("Please wait...", style=constants.LCD_CENT_JUST, line=3),
```

Comment:
What if style was a string? ("left", "center", "right")

Reply:
Sounds good to me, I'll implement this change